datasets.py

```diff
@@ -145,6 +145,17 @@ def load_labels(self) -> pd.DataFrame:
         return df
 
 
+class MalawiCorrectiveLabels2020(LabeledDataset):
+    def load_labels(self) -> pd.DataFrame:
+        Malawi_dir = raw_dir / "Malawi_corrective_labels_2020"
+        df = pd.read_csv(Malawi_dir / "Malawi_corrective_labels_2020.csv")
+        df.rename(columns={"latitude": LAT, "longitude": LON}, inplace=True)
+        df[CLASS_PROB] = (df["True value"] == 1).astype(int)
+        df[START], df[END] = date(2020, 9, 1), date(2021, 9, 1)
```

Comment:
@MsPixels for creating the dataset please use `df[START], df[END] = date(2020, 1, 1), date(2021, 12, 31)` this way we can use either February-February or September-September at training time

Reply:
Alright Ivan